src/utils/helper.rs

```diff
@@ -42,6 +44,37 @@ pub mod Util {
     pub const SGINFO: &str = "/usr/bin/sginfo";
     pub const JBOD_EXPORTER: &str = "/usr/bin/prometheus-jbod-exporter";
 
+    /// Returns an enum with true or false if a directory is empty
+    ///
+    /// This function verify if the directory is empty.
+    ///
+    /// # Arguments
+    ///
+    /// * `path` - a string reference
+    pub fn is_folder_empty(path: &str) -> io::Result<bool> {
+        Ok(fs::read_dir(path)?.take(1).count() == 0)
+    }
+
+    /// Verify if sysclass folder exists or otherwise exit and
+    /// provide software alternatives to jbod-rs.
+    pub fn verify_sysclass_folder(path: &str) {
+        if is_folder_empty(path).unwrap_or(false) {
+            println!(
+                "{} {} {}",
+                "==> ".red().bold(),
+                "jbod-rs not supported in this machine".bold(),
+                " <==".red().bold()
+            );
+            print!("{}", ":: ".bold().red());
+            println!("Use one of these alternatives: ");
+            println!(" - lsscsi");
+            println!(" - sas-lsi-tools");
+            println!(" - sas2ircu-status");
+            println!(" - sg3-utils");
+            exit(1);
```

Comment:
In the long run, would suggest being careful with `exit(1)`, it doesn't call `drop` implementations. (like in C/C++ it rely on `_atexit`.

If the program don't hold resource that need to be closed/cleaned it's "okay" but would suggest it's better to return `Result` or explore a crate like `anyhow` to get bubbling error without trouble.

Reply:
The program doesn't hold any resources.